bin/llvm2bpl.py

```diff
@@ -33,14 +33,20 @@ def llvm2bplParser():
                       help='turn on debug info')
   parser.add_argument('--mem-mod', dest='memmod', choices=['no-reuse', 'no-reuse-impls', 'reuse'], default='no-reuse-impls',
                       help='set the memory model (no-reuse=never reallocate the same address, reuse=reallocate freed addresses)')
+  parser.add_argument('--bit-vector', dest='bitvector', action="store_true", default=False,
+                      help='enable a bit-vector implemenation of SMACK')
+  parser.add_argument('--use-dsa', dest='usedsa', action="store_true", default=False,
```

Comment:
I would call this "infer-field-overlap" instead of "use-dsa" since we always use DSA for our memory model.